team/bundles/org.eclipse.jsch.ui/src/org/eclipse/jsch/internal/ui/preference/PreferencePage.java

```diff
@@ -305,6 +313,13 @@ private Control createKeyManagementPage(Composite parent){
 		gd.horizontalSpan=1;
 		keyLoad.setLayoutData(gd);
 
+		final Label keySizeValueLabel = new Label(group, SWT.NONE);
+		keySizeValueLabel.setText(Messages.CVSSSH2PreferencePage_148);
+		keySizeValue = new Spinner(group, SWT.BORDER);
+		keySizeValue.setValues(INITIAL_KEY_SIZE, MIN_KEY_SIZE, MAX_KEY_SIZE, MAX_KEY_SIZE_DIGIT_COUNT,
+				KEY_SIZE_INCREMENT, KEY_SIZE_INCREMENT);
```

Comment:
This can be structured a bit more to prevent an over-long line and two constants:
```suggestion
		int maxKeySize = Math.max(DSA_MAX_KEY_SIZE, RSA_MAX_KEY_SIZE);
		keySizeValue.setValues(INITIAL_KEY_SIZE, MIN_KEY_SIZE, maxKeySize, 0, KEY_SIZE_INCREMENT, KEY_SIZE_INCREMENT);
```